client/assets/css/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-image: url('images/homepage-image.png');
+    background-size: cover;
+    background-position: center;  
+    background-repeat: no-repeat;
+    min-height: 100vh;
+    background-attachment: fixed;
+    margin: 0;
+}
```

Comment:
When applying multiple background properties you can use the combined background property.
Definitely up to you whether you prefer declaring them individually or on one property.
Check out [this link](https://www.w3schools.com/cssref/css3_pr_background.asp) for more info.